streams/src/test/java/org/apache/kafka/streams/integration/LagFetchIntegrationTest.java

```diff
@@ -106,7 +108,7 @@ public void before() {
 
         consumerConfiguration = new Properties();
         consumerConfiguration.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, CLUSTER.bootstrapServers());
-        consumerConfiguration.setProperty(ConsumerConfig.GROUP_ID_CONFIG, name.getMethodName() + "-consumer");
+        consumerConfiguration.setProperty(ConsumerConfig.GROUP_ID_CONFIG, safeTestName + "-consumer");
```

Comment:
Somewhere else it is set as `"group-" + safeTestName`, is this change intentional?

Reply:
```suggestion
        consumerConfiguration.setProperty(ConsumerConfig.GROUP_ID_CONFIG, "group-" + safeTestName);
```

No, I just got tired of messing with every tests' idiosyncrasies.